src/main/io/transponder_ir.c

```diff
@@ -37,13 +38,35 @@
 
 #include "io/transponder_ir.h"
 
+#if 0
 PG_REGISTER_WITH_RESET_TEMPLATE(transponderConfig_t, transponderConfig, PG_TRANSPONDER_CONFIG, 0);
 
 PG_RESET_TEMPLATE(transponderConfig_t, transponderConfig,
     .provider = TRANSPONDER_ILAP,
     .reserved = 0,
     .data = { 0x12, 0x34, 0x56, 0x78, 0x9A, 0xBC, 0x0, 0x0, 0x0 }, // Note, this is NOT a valid transponder code, it's just for testing production hardware
 );
+#endif
+PG_REGISTER_WITH_RESET_FN(transponderConfig_t, transponderConfig, PG_TRANSPONDER_CONFIG, 0);
+
+void pgResetFn_transponderConfig(transponderConfig_t *transponderConfig)
+{
+    transponderConfig_t configTemplate = {
```

Comment:
For real? This sounds almost like 'broken by design' to me. After all, the developer is using `memcpy` for a reason...